R/drive_find.R

```diff
@@ -26,7 +26,7 @@
 #' By default, `drive_find()` does not include files in the trash: it adds `q =
 #' "trashed = false"` to the query. However, it will not do so if the user
 #' specifies a `q` search clause for trash inclusion or exclusion. To see only
-#' files in the trash, use [drive_view_trash()], which is a shortcut for
+#' files in the trash, use [drive_show_trash()], which is a shortcut for
```

Comment:
Overall, this is clearly going to be a great step forward re: permissions and featuring other info we excavate from `drive_resource`.

One nagging concern re: trash is that this is a row-based operation. We get the trash status, filter on it, and don't touch the columns. The path/publish/sharing versions are consistent: we feature some info in one or more new columns. 🤔